packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -140,6 +140,8 @@ export const DatePicker = forwardRef<HTMLElement, Props>(
 
         useClickOutside(wrapperRef, () => !state.calendarHidden && dispatch({ type: "TOGGLE" }));
 
+        useFocusOutside(wrapperRef, () => !state.calendarHidden && dispatch({ type: "TOGGLE" }));
```

Comment:
En potensiell forbedring er å trekke ut funksjonen for å toggle, siden den er den samme for både `clickOutside`og `focusOutside`